jest.config.js

```diff
@@ -1,20 +1,6 @@
 const baseConfig = require('kcd-scripts/jest')
 
 module.exports = {
-  collectCoverageFrom: baseConfig.collectCoverageFrom,
-  coverageThreshold: baseConfig.coverageThreshold,
-  projects: [
-    {
-      ...baseConfig,
-      displayName: 'jsdom',
-      testEnvironment: 'jest-environment-jsdom',
-    },
-    {
-      ...baseConfig,
-      displayName: 'node',
-      testEnvironment: 'jest-environment-node',
```

Comment:
Honestly, running all the exact same tests in `node` led to some poor testing practices and I didn't like the over-testing either. If someone really wants to have tests like this, then we can add a `other/node/__tests__/` directory with a `other/node/jest.config.js` file. I want the tests to be different.

Also, I don't think it's worthwhile to have tests for node. I'm pretty sure there's a very small number of people who benefit from this.